Directory.Build.props

```diff
@@ -1,7 +1,6 @@
 <Project>
 
   <PropertyGroup>
-    <LangVersion>preview</LangVersion>
```

Comment:
This is working around [Array.Contains no longer works in latest .NET 9 SDK (9.0.200) (npgsql/efcore.pg#3461)](https://github.com/npgsql/efcore.pg/issues/3461#issuecomment-2670681690)

Reply:
cc: @roji, @jaredpar